src/main/java/games/strategy/triplea/oddscalculator/ta/OddsCalculatorDialog.java

```diff
@@ -30,7 +30,7 @@ public static void show(final TripleAFrame taFrame, final Territory t) {
     dialog.addWindowListener(new WindowAdapter() {
       @Override
       public void windowClosed(final WindowEvent e) {
-        if (taFrame != null && taFrame.getUiContext() != null && !taFrame.getUiContext().isShutDown()) {
+        if (taFrame.getUiContext() != null && !taFrame.getUiContext().isShutDown()) {
```

Comment:
There's no explicit prior check here.  However, on line 28, `taFrame` is used to invoke the `getGame()` method, which will throw an NPE if `taFrame` is `null`.  Thus, static analysis says there's no need to check against `null` here because the anonymous `WindowAdapter` instance will never be created on line 30 if `taFrame` was `null` on line 28.